src/controllers/postControllers.ts

```diff
@@ -40,6 +42,37 @@ class PostControllers {
     }
   }
 
+  // COMMENT ENDPOINTS
+
+  async createComment(req: Request, res: Response, next: NextFunction) {
+    const { content }: { content: string } = req.body;
+    const { postId } = req.params;
+    try {
+      const post = await postService.findPostById(postId);
+      if (!post) {
+        return res.status(HttpStatusCodes.NOT_FOUND).json({ ok: false });
+      } else {
+        abilityService.for(req.user).throwUnlessCan('create', 'Comment');
+        await commentService.createComment(postId, req.user as User, content);
+        return res.status(HttpStatusCodes.CREATED).json({ ok: true });
+      }
+    } catch (error) {
+      return next(error);
+    }
+  }
+
+  async deleteComment(req: Request, res: Response, next: NextFunction) {
+    const { commentId } = req.params;
+    console.log('postController, deleteComment, commentID:', commentId);
```

Comment:
Let's remove this debug message, I think it's not necessary after development. Later on (or in a proper production environment) we might want to log what is happening on our server, and we might use some library for logging information to a log file. That is overkill for our needs here though.